FordLabs/PeopleMover
# Allow for edits of tag names that change case only

Co-authored-by: Elise Griffiths <egriff38@ford.com>
Co-authored-by: Vianney Willot <vianney.willot@gmail.com>

## Issue
Resolves #512

## What was done
- [x] No longer caring about name case when editing location or product tags

## How to test
1. Create a product tag
2. Edit product tag with a different case (e.g. Location -> LOCATION)
3. Create a location tag
4. Edit location tag with a different case (e.g. Product -> PRODUCT)
All of the above should happen successfully. Feel free to test roles as well. 

### FOR DEVS
Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/

Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### FOR APPROVAL
Approval URL: https://approvalbranchui.apps.pd01i.edc1.cf.ford.com/

Approval Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20ApprovalBranch/
